django_filters/rest_framework/filterset.py

```diff
@@ -36,3 +37,12 @@ def __init__(self, *args, **kwargs):
             helper.layout = Layout(*layout_components)
 
             self.form.helper = helper
+
+    @property
+    def qs(self):
+        from rest_framework.exceptions import ValidationError
```

Comment:
My understanding is that module imports are cached, so this should be fine. The problem is that the exceptions module eventually imports django's settings, which would prevent
```python
# settings.py
from django_filters import STRICTNESS

FILTERS_STRICTNESS = STRICTNESS.IGNORE
```
as it would create an import loop/error.